.github/CODEOWNERS

```
@@ -0,0 +1,8 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
+# review when someone opens a pull request.
+* rsandell
```

I think following the example in [the archetype](https://github.com/jenkinsci/archetypes/blob/ff60037cf37185fc33f2270946a8dd8e907e75fa/common-files/.github/CODEOWNERS) would be easier to maintain, since it relies on the centrally maintained group name rather than on individual GitHub usernames:

```suggestion
* @jenkinsci/ldap-plugin-developers
```